src/test/java/org/humancellatlas/ingest/security/AccountServiceTest.java

```diff
@@ -54,7 +59,9 @@ void success() {
 
             //and:
             var savedAccount = accountCaptor.getValue();
-            assertThat(savedAccount.getProviderReference()).isEqualTo(providerReference);
+            assertThat(savedAccount)
+                    .extracting("providerReference", "name")
```

Comment:
This breaks encapsulation of the Account class, and is brittle if renaming the provider/name fields. Assertions on the return values of the respective getters instead would address this

Reply:
`.extracting(Account::getProviderReference, Account::getName)`